recipe/meta.yaml

```diff
@@ -41,14 +38,11 @@ test:
     - test -f ${PREFIX}/include/lzma.h                          # [unix]
     - test -f ${PREFIX}/lib/pkgconfig/liblzma.pc                # [unix]
     - test -f `pkg-config --variable=libdir --dont-define-prefix liblzma`/liblzma${SHLIB_EXT}  # [unix]
```

Comment:
does it still produce .pc files if it's built with cmake? I guess so, or these tests would fail

Reply:
yes, it does, but maybe there should be similar tests for cmake here too